### ISSUE TITLE
File dump never asks for confirmation

### ISSUE THREAD

**Issue by angband-import** [angband/angband#3824]
**Reported by draconisPW on 2 Aug 2013 14:16 UTC**
Dumps use the get_file_text() method to build a file path. This method is supposed to check for file existence, but never does so because of the following error:

```
    /* Build the path */
    path_build(path, len, ANGBAND_DIR_USER, buf);

    /* Check if it already exists */
    if (file_exists(buf)) ...
```

It should be "path" instead of "buf".

**Comment by angband-import** [comment]
**Modified by takkaria on 7 Aug 2013 12:47 UTC**

**Comment by angband-import** [comment]
**Comment by takkaria on 30 Aug 2013 03:07 UTC**
Fixed in 2a941d6.
